go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/turbot/steampipe-plugin-slack
 go 1.18
 
 require (
-	github.com/slack-go/slack v0.7.4
+	github.com/slack-go/slack v0.11.2
```

Comment:
Are there any major breaking changes we have to handle in other tables in the bump from v0.7.4 to v0.11.2?